docs/usage.md

```diff
@@ -0,0 +1,190 @@
+---
+id: usage
+title: Usage Guide
+---
+
+There are quite a few tools in the Babel toolchain that try to make it easy for you to use Babel whether you're an "end-user" or building an integration of Babel itself. You can read more about the tools you decide to use in the "Usage" section of the docs.
+
+If you're building an app or library, there are two main things Babel can do for you:
+- Transform syntax
+- Polyfill features that are missing in your target environment
+- Code transformations (codemods)
+
+This short guide will show you how to compile your JavaScript application code that uses ES2015+ syntax into code that works in the last 2 versions of all browsers, as well as Safari versions 7 and up. That will involve both transforming new syntax and polyfilling missing features.
```

Comment:
I guess due to the whole "2 last versions" issue maybe we use another example? I'd rather just hardcode the browsers anyway personally (`chrome: 62`), shrug